QaMarsCompetition/QaMarsCompetition/PageObjects/ShareSkill.cs

```diff
@@ -76,19 +84,28 @@ public void ShareSkills(string title, string description, string credit)
             // identify skill exchange  and select credit
             skillexchange.Click();
             exchangecredit.SendKeys(credit);
```

Comment:
so you forgot to use the Excel reader, please retrieve data from excel sheet :)

Reply:
yes now i made the changes suggested by you. I used Excel data reader for retrieving data from excel sheet.